include/rocksdb/table_properties.h

```diff
@@ -284,9 +288,6 @@ struct TableProperties {
   UserCollectedProperties user_collected_properties;
   UserCollectedProperties readable_properties;
 
-  // The offset of the value of each property in the file.
-  std::map<std::string, uint64_t> properties_offsets;
```

Comment:
Note to reviewer:
I wonder if we need to further discuss the impact of this change to non-fb use case since `TableProperties::properties_offsets` is a public API.